RoosterPlanner.Api/Startup.cs

```diff
@@ -81,6 +81,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<ITaskService, TaskService>();
             services.AddTransient<IShiftService, ShiftService>();
             services.AddTransient<IBlobService, BlobService>();
```

Comment:
Deze service (en misschien andere) zou naar AddScoped kunnen.

Reply:
Laten we hier even mondeling over praten